html/comparison.html.haml

```diff
@@ -240,8 +238,7 @@
                 %span.summary-part{"ng-show" => "diff.users.only_in2"}
                   {{diff.meta.description_b}}: {{diff.users.only_in2.length}} users
                 %span.summary-part{"ng-show" => "diff.users.common"}
-                  %a{:class=>"show-common-elements", href: "#users_both"}
-                    both
+                  %a{:class=>"show-common-elements", href: "#users_both"}both
```

Comment:
Avoid defining `class` in attributes hash for static class names<br>Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace